backend/src/types/external-apis.ts

```diff
@@ -0,0 +1,7 @@
+export interface BookQueryResponse {
```

Comment:
Suggestion: this could be better of in the book module in case it is not used anywhere else.